config/200-clusterrole.yaml

```diff
@@ -60,6 +60,7 @@ metadata:
   labels:
     serving.knative.dev/release: devel
     serving.knative.dev/controller: "true"
+    serving.knative.dev/network-provider: istio
```

Comment:
Perhaps: `networking.knative.dev/ingress-provider: istio`?

I suspect that we will also end up with `networking.knative.dev/tls-provider: certmanager` :)

Reply:
cc @ZhiminXiang

Reply:
That sgtm.  Done.